catalyst/dl/utils/torch.py

```diff
@@ -96,7 +96,48 @@ def get_loader(
     return loader
 
 
+def process_model_params(
```

Comment:
what do you think about `group_model_params` name?
or `gather_*` / `arrange_*`

btw, catalyst naming `process_something` suggests inplace change of the args. kwargs, rather then returning some result (for methods with result, it's better to use `get_*`)

Reply:
i'll name it `get_*` then